ultraplot/axes/plot.py

```diff
@@ -3147,6 +3173,87 @@ def linex(self, *args, **kwargs):
         """
         return self.plotx(*args, **kwargs)
 
+    def _apply_lollipop(self, *args, horizontal=False, **kwargs):
+        linecolor = kwargs.pop("linecolor", None)
+        linecolor = _not_none(linecolor, rc["lollipop.linecolor"])
+        linestyle = kwargs.pop("linestyle", None)
+        linestyle = _not_none(linestyle, rc["lollipop.linestyle"])
+        size = kwargs.pop("size", None)
```

Comment:
We should use markersize here to be consistent with other parts of the matplotlib api.